src/index.html

```diff
@@ -9,6 +9,12 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="wrapper">
```

Comment:
You don't need to wrap the content in an extra `div` block, you can set a `wrapper` class for the `body` and customize it.